jekyll/_style/style-voice.adoc

```diff
@@ -0,0 +1,34 @@
+= Style and voice
+:page-layout: classic-docs
+:icons: font
+:toc: macro
+:toc-title:
+
+== Talk confidently and directly
+Use an active voice, talk directly to the reader using simple, direct, clear and confident language. Using active voice helps us to keep instructions as short as possible, and is, in most cases, the quickest way to convey meaning to the reader
+
+* **Good**: Click **Go** and your project will complete.
+* **Bad**: When you click the **Go** button your project should complete.
+
+== Write for anyone and everyone
+Assume technical competence but explain concepts clearly and simply. Some concepts we write about are complex, and outside a non-technical reader's experience, but the flow of CircleCI docs should be accessible to anyone.
+
+== Word choices
+* Steer clear of making assumptions about the complexity of a process. Avoid descriptions like "easy", "easily" or "simply".
+* When specifying a tool or way of performing an action, use "using" instead of "via". The latter is harder on non-English speakers.
+* No whitelist/blacklist. Instead use safelist/blocklist.
+* No master/slave. Instead use leader/follower.
+
```

Comment:
I don't intend to ban all of them, but can we add "be careful not to be ambiguous when using "should, should be""? [Google's style guide](https://developers.google.com/style/word-list#letter-s) also mentions this, and I sometimes see translators mistranslate sentences like this: "A cache saved on one branch will be used by jobs run on other branches so they should only be used for data that is suitable to share across branches."

Reply:
@Mayuko-Sa Great call out here.